RealmTasks Android/app/src/main/java/io/realm/realmtasks/TaskActivity.java

```diff
@@ -184,21 +181,16 @@ public boolean onClicked(ItemViewHolder viewHolder) {
         @Override
         public void onChanged(ItemViewHolder viewHolder) {
             adapter.onItemChanged(viewHolder);
-            adapter.notifyItemChanged(viewHolder.getAdapterPosition());
         }
 
         @Override
         public void onAdded() {
             adapter.onItemAdded();
-            adapter.notifyItemInserted(0);
```

Comment:
When I added `adapter.notifyItemInserted(0);` again, it encountered an exception like the following:

```
FATAL EXCEPTION: main
                                                                     Process: io.realm.realmtasks, PID: 20759
                                                                     java.lang.IndexOutOfBoundsException: Inconsistency detected. Invalid view holder adapter positionViewHolder{adbf301 position=4 id=-1, oldPos=2, pLpos:2 scrap [attachedScrap] tmpDetached no parent}
                                                                         at android.support.v7.widget.RecyclerView$Recycler.validateViewHolderForOffsetPosition(RecyclerView.java:5297)
                                                                         at android.support.v7.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:5479)
                                                                         at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:5440)
                                                                         at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:5436)
                                                                         at android.support.v7.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:2224)
                                                                         at android.support.v7.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1551)
                                                                         at android.support.v7.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1511)
                                                                         at android.support.v7.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:595)
                                                                         at android.support.v7.widget.RecyclerView.dispatchLayoutStep1(RecyclerView.java:3534)
                                                                         at android.support.v7.widget.RecyclerView.dispatchLayout(RecyclerView.java:3310)
                                                                         at android.support.v7.widget.RecyclerView.onLayout(RecyclerView.java:3844)
                                                                         at android.view.View.layout(View.java:16630)
                                                                         at android.view.ViewGroup.layout(ViewGroup.java:5437)
                                                                         at android.widget.RelativeLayout.onLayout(RelativeLayout.java:1079)
                                                                         at android.view.View.layout(View.java:16630)
                                                                         at android.view.ViewGroup.layout(ViewGroup.java:5437)
                                                                         at android.widget.FrameLayout.layoutChildren(FrameLayout.java:336)
                                                                         at android.widget.FrameLayout.onLayout(FrameLayout.java:273)
                                                                         at android.view.View.layout(View.java:16630)
                                                                         at android.view.ViewGroup.layout(ViewGroup.java:5437)
                                                                         at android.support.v7.widget.ActionBarOverlayLayout.onLayout(ActionBarOverlayLayout.java:437)
                                                                         at android.view.View.layout(View.java:16630)
                                                                         at android.view.ViewGroup.layout(ViewGroup.java:5437)
                                                                         at android.widget.FrameLayout.layoutChildren(FrameLayout.java:336)
                                                                         at android.widget.FrameLayout.onLayout(FrameLayout.java:273)
                                                                         at android.view.View.layout(View.java:16630)
                                                                         at android.view.ViewGroup.layout(ViewGroup.java:5437)
                                                                         at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1743)
                                                                         at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1586)
                                                                         at android.widget.LinearLayout.onLayout(LinearLayout.java:1495)
                                                                         at android.view.View.layout(View.java:16630)
                                                                         at android.view.ViewGroup.layout(ViewGroup.java:5437)
                                                                         at android.widget.FrameLayout.layoutChildren(FrameLayout.java:336)
                                                                         at android.widget.FrameLayout.onLayout(FrameLayout.java:273)
                                                                         at com.android.internal.policy.PhoneWindow$DecorView.onLayout(PhoneWindow.java:2678)
                                                                         at android.view.View.layout(View.java:16630)
                                                                         at android.view.ViewGroup.layout(ViewGroup.java:5437)
                                                                         at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:2171)
                                                                         at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1931)
                                                                         at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1107)
                                                                         at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6013)
                                                                         at android.view.Choreographer$CallbackRecord.run(Choreographer.java:858)
                                                                         at android.view.Choreographer.doCallbacks(Choreographer.java:670)
                                                                         at android.view.Choreographer.doFrame(Choreographer.java:606)
                                                                         at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:844)
                                                                         at android.os.Handler.handleCallback(Handler.java:739)
                                                                         at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                         at android.os.Looper.loop(Looper.java:148)
                                                                         at android.app.ActivityThread.main(ActivityThread.java:5417)
                                                                         at java.lang.reflect.Method.invoke(Native Method)
                                                                         at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
                                                                         at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```

I think just adding `adapter.notifyDataSetChanged();` for just this PR. Since this PR's purpose is for updating its libraries.